### ISSUE TITLE
supervised training optimization for best performance

### ISSUE THREAD

**Issue by taeil** [Berkeley-Data/irrigation_detection#40]
- augmentation 
- pre-train model (e.g., imagenet)
- learning rate 
- validation metric (AUC) 
- a lot more parameters

**Comment by TsungChinHanKen** [comment]
A quick check, should we request additional credits for aws from prof recently?

**Comment by taeil** [comment]
Yes we should.

**Comment by TsungChinHanKen** [comment]
1. current supervised was experimented with given dataset. update parameters is related to pull request 'activation function' and 'tensorflow addon metrics'

2. The project direction is subject to change and still TBD, we may instead using other sota benchmark for the project, or revisit this items as per request.
